rds-deployment-ha-gateway/azuredeploy.json

```diff
@@ -8,13 +8,20 @@
         "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
       }
     },
-    "loadBalancer": {
+    "gatewayLoadBalancer": {
       "type": "string",
       "metadata": {
-        "description": "The Loadbalancer name must match from the RDS deployment. And default value taken by template is loadbalancer."
+        "description": "The gateway Load balancer name must match from the RDS deployment. And default value taken by template is loadbalancer."
       },
       "defaultValue": "loadBalancer"
     },
+    "gatewayPublicip": {
```

Comment:
nitpicking here, 
but want to see "Ip" starting with capital too, i.e. "gatewayPublicIp"